setup.py

```diff
@@ -24,15 +24,6 @@
           "PyTango>=9.2.2",
           "numpy",
           "jsonschema"],
-      tests_require=[
```

Comment:
How come is the 'tests_require' removed? Is it unneeded?

Reply:
See: https://github.com/ska-sa/tango-simlib/pull/105#discussion_r353102050

Reply:
Don't we still need it if we are just running nose tests in the module?

Reply:
Ideally what was listed here is listed under dependencies in the tox file. and tox should be able to invoke nose tests directly.

Reply:
We need to re-instate this since we are not going with tox.